editor/plugins/org.fusesource.ide.camel.editor/src/org/fusesource/ide/camel/editor/internal/l10n/messages.properties

```diff
@@ -52,6 +52,7 @@ miscellaneousDrawerTitle=Miscellaneous
 
 editorPreferencePageDescription=Preferences used when creating Enterprise Integration Patterns...
 editorPreferencePageDefaultLanguageSetting=The default language used for expressions in EIPs
+editorPreferencePagePreferIdAsLabelSetting=If enabled the ID values will be used for labels if not overriden by preferred labels 
```

Comment:
Having 2 if, one at the beginning and one at the end seems to complexify the sentence.
What about "If enabled and not overridden by preferred labels, the ID values will be used for labels"?

@bfitzpat @MelissaFlinn The advice from a native speaker is welcome :-)

Reply:
I recommend: "Use ID values for all component labels (overridden by preferred labels)."
and then for the Preferred Labels option change to: "Select preferred labels for specific components (except endpoints)."

Reply:
IMHO I recommend keeping this simpler - "Use ID values for labels if not overridden by preferred labels" The "if enabled" is suggested by the checkbox itself, so it's not needed. @MelissaFlinn may have other ideas. :)